README.md

```diff
@@ -1,462 +1,259 @@
-# DocArray v2
+# DocArray - Version 2
 
-this repo is a PoC for the new version of DocArray. The scope of the PoC is twofolds:
+_**NOTE**: This readme refers to version 2 of DocArray, a rewrite that is currently at alpha stage.
```

Comment:
maybe not refer directly to readme as it will be displayed in the documentation as well ?